### PR TITLE
WSTEAM1-312: Add style changes to embed error message component

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves JIRA [WSTEAM1-312](https://jira.dev.bbc.co.uk/browse/WSTEAM1-312)

Overall changes
======
Updates the error link underline to black, and the error message colour to grey. Updates A11y following swarm.

Code changes
======
- _Updates the `EmbedError` styling._

Testing
======
1. _Run simorgh locally (using yarn dev)_
2. _Go to: http://localhost:7080/pidgin/articles/cqwq7dm61zeo_

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by louisearchibald — APPROVED** [review]

**Review by Isabella-Mitchell — APPROVED** [review]
Checked locally on the link (http://localhost:7080/pidgin/articles/cqwq7dm61zeo.amp) and compared to the design.

I noticed that there are no hover/ focus styles outlined in the design, so I assume the desired behaviour is that the link falls back to the default inline link styles. (Which it does.)

**Review by Isabella-Mitchell — CHANGES_REQUESTED** [review]
Sorry I just realised the design also outliens fontSizes that I can't see in the styles. (Correct me if I'm wrong!)

Screenshot 2023-08-24 at 10 16 17

I wonder if we also need to add `...fontSizes.longPrimer` and `...fontSizes.bodyCopy` like below

```
  errorMessage: ({ fontVariants, fontSizes, palette }: Theme) =>
    css({
      color: palette.GREY_10,
      ...fontVariants.sansRegular,
      ...fontSizes.longPrimer,
    }),
  inlineLink: ({ fontSizes, palette, fontVariants }: Theme) =>
    css({
      color: palette.BLACK,
      borderBottom: `1px solid black`,
      ...fontVariants.sansRegular,
      ...fontSizes.bodyCopy,
    }),
```

**Review by amoore108 — APPROVED** [review]

File: src/app/components/Embeds/EmbedError/index.styles.tsx [hunk 1]
```diff
@@ -19,11 +19,19 @@ const styles = {
       ...fontVariants.sansBold,
       paddingTop: `${spacings.FULL}rem`,
     }),
-  errorMessage: ({ fontVariants }: Theme) =>
+  errorMessage: ({ fontVariants, palette, fontSizes }: Theme) =>
     css({
+      color: palette.GREY_10,
       ...fontVariants.sansRegular,
+      ...fontSizes.longPrimer,
```
Comment (amoore108) [inline src/app/components/Embeds/EmbedError/index.styles.tsx]: May be something we can add later, but we have a `<Text />` component that is polymorphic, so you can set it to any valid HTML element as well as specifying the font variant and size, e.g:
```
    <Text as="strong" fontVariant="sansRegular" size="longPrimer">
      {message}
    </Text>
```

Just saves having to set the font styling in CSS.

Comment (Isabella-Mitchell) [reply src/app/components/Embeds/EmbedError/index.styles.tsx]: Oh yh good spot!

**Review by Isabella-Mitchell — APPROVED** [review]

**Pull request merged**